apps/backend/user-service/src/app.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
+
+  @Get()
+  getHello(): string {
```

Comment:
Add annotation to the endpoint and install swagger

https://docs.nestjs.com/openapi/introduction